core/src/main/java/org/elasticsearch/index/mapper/core/TypeParsers.java

```diff
@@ -197,9 +197,6 @@ public static void parseField(FieldMapper.Builder builder, String name, Map<Stri
             } else if (propName.equals("index")) {
                 parseIndex(name, propNode.toString(), builder);
                 iterator.remove();
-            } else if (propName.equals("tokenized")) {
```

Comment:
The builder method is still used by the string parser, which sets tokenized = true in the analysed case